docs/cell_census_storage_and_release_policy.md

```diff
@@ -1,45 +1,48 @@
 # Cell Census storage & release policy
 
-**Last edited**: Dec, 2022.
+**Last edited**: March, 2023.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in [BCP 14](https://tools.ietf.org/html/bcp14), [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
 
----
+## Definitions
 
-The cell census MUST be stored in the following S3 root bucket:
+* **Cell Census build**: a SOMA collection with the Cell Census data [as specified in the Cell Census schema](https://github.com/chanzuckerberg/cell-census/blob/main/docs/cell_census_schema.md#data-encoding-and-organization). 
+* **Cell Census source H5AD files**: the set of H5AD files used to create a Cell Census build.
+* **Cell Census release**: a Cell Census build that is publicly hosted online.
+* **Cell Census release `tag`**:  a label for a Cell Census release, it MUST be a string of printable ASCII characters.
 
-`s3://cellxgene-data-public/`
+## Cell Census data storage policy
 
-All contents of a Cell Census build data MUST be deposited in a folder named `cell-census`: 
+The following S3 bucket MUST be used as the root to store Cell Census data:
 
-`./cell-census/`
+`s3://cellxgene-data-public/`
 
-This folder MUST contain all public builds and releases defined as:
+Cell Census data MUST be deposited under a folder named `cell-census` of the root S3 bucket:
+ 
+ `./cell-census`
+ 
+All data related to a Cell Census **release** MUST be deposited in a folder named with the **tag** of the release:
 
-* Census "build" is a SOMA collection / container containing cell data.
-* Census "RELEASE" is a "build" which has been officially blessed and released in a formal manner.
-* A "build" is named with a `tag`, which is a string of printable ASCII (eg, "foo", "2022-11-01", "1-rc2.9"):
-   `./cell-census/[tag]/`
-* a RELEASE is named with a tag following the conventional format `release-[counter]`:
-   `./cell-census/release-[counter]/`
+ `./cell-census/[tag]/`
 
-The Cell Census SOMA-related files MUST be deposited within the release folder on a folder named `soma`:
+The Cell Census **release** MUST be deposited in a folder named `soma`:
 
-`./cell-census/release-[counter]/soma/`
+`./cell-census/[tag]/soma/`
 
-All h5ads used to create the Cell Census MUST be copied within the release folder into a folder named `h5ads`:	
-`./cell-census/release-[counter]/h5ads/`
+All Cell Census **source h5ads** used to create a specific Cell Census **release** MUST be copied into a folder named `h5ads`:	
+`./cell-census/[tag]/h5ads/`
```

Comment:
I think the larger issue is, "can we--and how might we--change the S3 layout if we need to do so?" E.g., if we want to change the releases.json structure or the folder structure, that's fine for those using the latest API package. But since we're intending for older releases to be accessible using _older_ API packages, we're probably forced to keep older S3 layout and data in place while introducing a new S3 layout in parallel. (Newer APIs can decide to be backwards-compatible with the older layout, or simply ignore the older releases, leaving those visible via older API packages).

All that said, since this file is versioned in git I'm okay with it having "forever" language. My take is that it's only promising that API packages will adhere to the specs of this file as it stands at the time of the build.